.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
+    "quotes": [2, "single"]
```

Comment:
What does the "2" mean? I looked in [the docs](http://eslint.org/docs/rules/quotes), but it didn't seem to explain it (it just used it in the example). Should we also include `"avoid-escape"`? That seems like a pretty reasonable option.